### ISSUE TITLE
Vulnerability "Running container images should have vulnerability findings resolved" no longer scanned

### ISSUE THREAD

**Issue by chudytom** [Azure/AKS#2965]
**Describe the bug**
Before when looking at the security for AKS clusters I was able to get all the information for a vulnerability named "Running container images should have vulnerability findings resolved". It produced lots of useful data on which cluster, specific containers run. Now, this option has vanished. I cannot see in the Azure Portal under security recommendations. I cannot also see it when filtering the results in the Azure Resource Graph queries under 'microsoft.security/assessments'

**To Reproduce**

1. Go to 'Azure Resource Graph Explorer'
2. Run the following query
```
securityresources
| where ['type'] == 'microsoft.security/assessments'
| where tostring( properties.displayName) == 'Running container images should have vulnerability findings resolved'
``` 

**Expected behavior**
Results for "Running container images should have vulnerability findings resolved" vulnerability should be visible

All I can see is "Container registry images should have vulnerability findings resolved" where only static images in an ACR are scanned. They do not contain information about where the workloads actually run

**Comment by t0p4** [comment]
I ran into the same problem, I checked all the official deployment documents, but I got no results for running images in aks

**Comment by tspivakms** [comment]
@chudytom @t0p4 can you please both create support tickets so our engineering team can have additional context for investigation?

**Comment by chudytom** [comment]
@tspivakms I just created the support request. Here is the Support request ID 2206070050002273

**Comment by tspivakms** [comment]
Thanks @chudytom I expect that your issue would be or was addressed by the support ticket. If that is not the case, please let us know here.

**Comment by chudytom** [comment]
Yes, the product team is in the process of investigation. They are happy to take the case as based on the responses in this issue it affects more customers. 

I will share the findings here once I know the root cause.

**Comment by tspivakms** [comment]
Thanks @chudytom.

**Comment by chudytom** [comment]
@tspivakms my case was resolved. I got the information from the backend team that there was some error in the implementation and it's now been fixed internally.
I tested it and it seems to be working from my side.
However with my support plan  I was not notified about the Root Cause Analysis for the issue. Maybe you are able to get more info from the team and share here with the community. TrackingID#2206070050002273
